src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -90,7 +90,9 @@ def arch_repo_table_cell(repo, arch, package_name)
                         { title: link_title, rel: 'nofollow' }
                        )
     end
-    result += '</td>'.html_safe
+
+    status_desc = Webui::PackageHelper.get_package_status_description(status['code'])
+    result += " <img title='#{status_desc}' class='icons-help' alt='#{status_desc}' src='/assets/s.gif' /></td>".html_safe
     result
```

Comment:
This line is not needed... so if you want to get rid of it, just remove it :smile: